### PR TITLE
CI configured for 2023b branch 

### PR DESCRIPTION
**Pull request by dibryant**
Fixes for #135 
https://github.com/opendatahub-io/notebooks/issues/135#issue-1782677134

Added Configuration for 2023b branch.

### CODE REVIEW

**Comment by dibryant** [comment]
make ci-operator-config

File: ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-2023b.yaml [hunk 1]
```diff
@@ -0,0 +1,416 @@
...
+      IMAGE_REPO: workbench-images
+      IMAGE_TAG: YearIndex
+      RELEASE_VERSION: cuda-rstudio-c9s-python-3.9-2023b
+    workflow: opendatahub-io-ci-image-mirror
+  zz_generated_metadata:
+   branch: 2023b
+   org: opendatahub-io
+-  repo: notebooks
++  repo: notebooks
+\ No newline at end of file
```
Comment (harshad16) [inline ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-2023b.yaml:416]:
```suggestion

```

This seems to be added by the editor, this shouldn't be here.

**Review by harshad16 — CHANGES_REQUESTED** [review]
lgtm
seems like some other parts were left out.

**Review by atheo89 — COMMENTED** [review]
Hi, I've noticed that the 2023b branch is currently out of sync with the latest commits. It seems that the build and test instructions for `runtimes/minimal/ubi8-python-3.8` and `runtimes/minimal/ubi9-python-3.9` are missing from the config file. To address this, could you perform a fetch and rebase? This would ensure that the branch incorporates the most recent changes.

**Comment by dibryant** [comment]
pj-rehearse

**Review by harshad16 — APPROVED** [review]
/lgtm
/approve
Thanks :100:

**Pull request merged**